### PR TITLE
Added the missing any and size_t types

### PR DESCRIPTION
**Pull request by pouyakary**
The types `size_t` and `any` were not defined in the syntax so this patch adds them:

<br>

Screen Shot 1400-05-02 at 18 08 45

### CODE REVIEW

**Comment by danieldaeschle** [comment]
any is not needed imo.
Should be removed from V

File: syntaxes/v.tmLanguage.json [hunk 1]
```diff
@@ -267,7 +267,7 @@
 							"name": "meta.expr.numeric.cast.v"
 						},
 						{
-							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune)(?=\\s*\\()",
+							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune|size_t)(?=\\s*\\()",
```
Comment (danieldaeschle) [inline syntaxes/v.tmLanguage.json:270]: Can you remove ustring please? It was removed recently.

**Review by danieldaeschle — CHANGES_REQUESTED** [review]

File: syntaxes/v.tmLanguage.json (hunk 1 above)
Comment (pouyakary) [reply syntaxes/v.tmLanguage.json:270]: Yeah sure.

Comment (danieldaeschle) [reply syntaxes/v.tmLanguage.json:270]: Thanks :)

**Comment by pouyakary** [comment]
I am going to implement the #246  and #247, before doing so I also added the [CSpell](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) and fixed two little spellings.

**Comment by danieldaeschle** [comment]
Running cspellchecker is fine but please remove it from the project

**Comment by danieldaeschle** [comment]
Please implement the other things in extra PRs that i can merge this

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (danieldaeschle) [inline .vscode/settings.json:1]: Remove this file

File: .vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (danieldaeschle) [inline .vscode/extensions.json:1]: Remove this file

**Review by danieldaeschle — CHANGES_REQUESTED** [review]

**Comment by pouyakary** [comment]
Sure do

**Pull request merged**